mslib/mscolab/app/__init__.py

```diff
@@ -47,7 +47,7 @@
 APP.config.from_object(__name__)
 APP.route = prefix_route(APP.route, SCRIPT_NAME)
 
-APP.config['MSCOLAB_DATA_DIR'] = mscolab_settings.MSCOLAB_DATA_DIR
+APP.config['MSCOLAB_DATA_DIR'] = mscolab_settings.FILE_DATA
```

Comment:
Shouldn't the key used with APP.config be updated as well?

Reply:
Actually, why do we even have two global configuration places instead of just one?

Reply:
Do you mean moving the APP.config[*] to the users scope?

Reply:
Question 1 is: should it be `APP.config['FILE_DATA']` instead?

Question 2 is: why do we have both `APP.config` and `mscolab_settings`?

Reply:
1. yes :)
2. the idea was to have a different layer for users than we have in the installed system.  
In my other projects we used this approach and I copied it to MSS too. It is a python script because it is also used to add functionality by the user. For mscolab we could check if a yaml is an option. For mswms it is needed to add different dataaccess or plots. At the time we created the configuration option we wanted to have it similar to mswms.

Reply:
We can discuss that here, but it then lost after this is closed.